packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -71,12 +76,24 @@ Conditions:
           - Ref: FileAssetsBucketName
   CreateNewKey:
     Fn::Equals:
-      - ''
+      - 'BOOTSTRAP_KEY'
       - Ref: FileAssetsBucketKmsKeyId
+  CreateNewEcrKey:
+    Fn::Equals:
+      - 'BOOTSTRAP_KEY'
+      - Ref: ImageAssetsEcrKmsKeyId
+  CreateCustomerMasterKey:
+    Fn::Or:
+      - !Condition CreateNewKey
+      - !Condition CreateNewEcrKey
   UseAwsManagedKey:
     Fn::Equals:
-      - 'AWS_MANAGED_KEY'
+      - 'DEFAULT_ENCRYPTION'
```

Comment:
Can't change these.

Reply:
As mentioned above as long as we also change the inputs to the template this should not be a problem but in the end make it easier to understand what is happening